documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -346,6 +346,12 @@ include::{partialsdir}/modules/all-connectors/proc-triggering-an-incremental-sna
 
 include::{partialsdir}/modules/all-connectors/proc-stopping-an-incremental-snapshot-nosql.adoc[leveloffset=+3]
 
+// Type: concept
+[id="mongodb-blocking-snapshots"]
+==== Blocking snapshots
+
+include::{partialsdir}/modules/all-connectors/con-connector-blocking-snapshot.adoc[leveloffset=+3]
+
```

Comment:
If blocking snapshots are not supported downstream for MongoDB, conditionalize this section as community content and remove the comment for the Nebel processor (`//Type: concept`).

```suggestion
ifdef::community[]
[id="mongodb-blocking-snapshots"]
==== Blocking snapshots

include::{partialsdir}/modules/all-connectors/con-connector-blocking-snapshot.adoc[leveloffset=+3]

endif::[]

```

Reply:
@jpechane Are we supporting it for downstream?

Reply:
@jpechane gentle ping.

Reply:
@mfvitale I suppose we are going to support it fro all connectors. Are there any technical issues?

Reply:
Yes we are supporting (Db2,MongoDB ,MySQL,Oracle,PostgreSQL,SQL Server)

Reply:
:+1: